azext_edge/tests/helpers.py

```diff
@@ -62,6 +63,9 @@ def find_extra_or_missing_names(
         else:
             error_msg.append(msg)
     missing_names = [name for name in expected_names if name not in result_names]
+    # ignore certain name prefixes
+    for prefix in ignore_prefixes or []:
+        missing_names = [name for name in missing_names if not name.startswith(prefix)]
```

Comment:
would this also apply to extra?

Reply:
yes, but - after chatting offline I think it's best to remove this kind of... plumbing... until we have a better idea of where/how to utilize it.